typescript/packages/subsurface-viewer/src/SubsurfaceViewer.tsx

```diff
@@ -99,6 +99,9 @@ export interface SubsurfaceViewerProps {
      */
     isLoadedCallback?: (arg: boolean) => void;
 
+    onDragStart?: (info: unknown, event: unknown) => void;
```

Comment:
We should try to type the arguments.
info is a PickingInfo, right ?
event is burried too deep to have really typing info :(